extensions/ql-vscode/src/extension.ts

```diff
@@ -771,7 +771,7 @@ async function activateWithInstalledDistribution(
     ) =>
       databaseUI.handleChooseDatabaseLgtm(progress, token),
       {
-        title: 'Adding database from LGTM. Choose a language from the dropdown, if requested.',
+        title: 'Adding database from LGTM.',
```

Comment:
```suggestion
        title: 'Adding database from LGTM',
```

Reply:
Thanks for the feedback. I was wondering about whether to keep this in as well. I think it looks better without it.